monai/handlers/tensorboard_handlers.py

```diff
@@ -258,10 +258,10 @@ def __init__(
             interval: plot content from engine.state every N epochs or every N iterations, default is 1.
             epoch_level: plot content from engine.state every N epochs or N iterations. `True` is epoch level,
                 `False` is iteration level.
-            batch_transform: a callable that is used to transform the
-                ``ignite.engine.batch`` into expected format to extract several label data.
-            output_transform: a callable that is used to transform the
-                ``ignite.engine.output`` into expected format to extract several output data.
+            batch_transform: a callable that is used to construct `[images, labels]` pair from `engine.state.batch`.
```

Comment:
my suggestion is:
batch_transform: a callable that is used to transform the output batch achieved from `ignite.engine.state.batch` to construct `[images, labels]` pair ...

Reply:
updated in the latest commit.
Thanks.